ntbs-service/Pages/Shared/_NotificationLayoutWithAlertsAndSideMenu.cshtml

```diff
@@ -93,11 +93,14 @@
                             Previous History
                         </a>
                     </li>
-                    <li class="@("contents-list-item " + (currentPage == NotificationSubPaths.EditMDRDetails ? "contents-list-item--current" : ""))">
-                        <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="@RouteHelper.GetNotificationPath(NotificationSubPaths.EditMDRDetails, Model.NotificationId)">
-                            MDR Details
-                        </a>
-                    </li>
+                    @if(Model.Notification?.ClinicalDetails?.IsMDRTreatment == true) //TODO-384 drug resistance profile check
```

Comment:
ditto

Also all these places are running the check and will have to double the check up. should we abstract into a method that gets called? Perhaps just wants to be done by NTBS-384...